### ISSUE TITLE
Ordering of events and exhibitions

### ISSUE THREAD

**Issue by jamesgorrie** [wellcometrust/wellcomecollection.org#2972]
On the homepage and what's on.

Combination of exhibitions and events (homepage, today, and this weekend):
* currentTemporaryExhibitionPromos
* eventPromos
* upcomingTemporaryExhibitionPromos
* permanentExhibitionPromos

Exhibitions and events split (everything view):
* Exhibitions
  * currentTemporaryExhibitionPromos
  * permanentExhibitionPromos
  * upcomingTemporaryExhibitionPromos
  * eventPromosSplitAcrossMonths

**Comment by Heesoomoon** [comment]
It will be best to group similar things together i.e. Temporary and Permanent exhibitions. Also, this will make the permanent exhibitions more prominent which are more relevant to the new visitors.

Here's my proposal for the ordering based on the type of things/scale of our programme.
**Combination of exhibitions and events**
This week (homepage), today, and this weekend:
- Current temporary exhibition
- Permanent exhibition
- Upcoming temporary exhibition
- Installation (hide delisted i.e installations from SIB)
- Events (upcoming then daily tour)

**Split view **
Exhibitions
- Current temporary exhibition
- Permanent Exhibition
- Upcoming temporary exhibition

Events
- Installation (hide delisted i.e installations from SIB)
- Events (upcoming then daily tour)

@jamesgorrie Let's confirm with @jennpb and Jen S if they had reasons for displaying Permanent Exhibitions last in the combined list.

**Comment by hthair** [comment]
I agree that upcoming exhibitions should be after current and permanent. From what I've seen I feel this is the way people browse, expecting to see upcoming things after what they can currently do.

**Comment by jamesgorrie** [comment]
@Heesoomoon could we not consolidate this into two lists then, as that's pretty much what we have i.e.

# Exhibitions
Current temporary exhibition
Permanent Exhibition
Upcoming temporary exhibition
Installation (hide delisted i.e installations from SIB)

# Events
Events (upcoming then daily tour)

Or consolidate the other way round of not being split?

**Comment by Heesoomoon** [comment]
@jamesgorrie Yes, we can consolidate if that logic makes sense.

**Comment by jennpb** [comment]
That sounds good to me. We didn't have any reasons for putting Perm Exhibitions at the end of the list.

**Comment by jamesgorrie** [comment]
> We didn't have any reasons for putting Perm Exhibitions at the end of the list.

It was definitely a decision that was made as it's been explicitly coded that way - can't seem to find where though... odd.

**Comment by hthair** [comment]
might have been to surface newer content first?

**Comment by jennpb** [comment]
Ah, yes, you might be right. The 2 permanent exhibitions will get in the way of more urgent cards - events this week that might require action/booking. 

In that case, here's the decision: can we keep permanent exhibitions at the end of the list for now? @Heesoomoon - thoughts?

**Comment by jamesgorrie** [comment]
We know that the point of these pages is to get people navigating to events or exhibitions.
I can set this up as a test if we want, and then we can iterate on it with results helping lead the way?

**Comment by pollecuttn** [comment]
@jennpb to decide about the ordering

**Comment by jamesgorrie** [comment]
> @jennpb to decide about the ordering

Or a test?

**Comment by jennpb** [comment]
Can you please suggest a test? I don't know what you're trying to find out.

We need to be able to take action on #2971 by 24 June, so a test can't block doing that.
